### PR TITLE
Implements the `build` function

### LINKED ISSUE #7: Implement `build` method
Implement a `build` method for the `ContinuousIntegrationServer` that builds a cloned repository.

### PR DESCRIPTION
**Pull request by simonblo**
Implements the build function as described in issue #7.

### CODE REVIEW

File: src/main/java/com/ci/ContinuousIntegrationServer.java [hunk 1]
```diff
@@ -77,8 +78,14 @@ private void cloneRepo() {
...
     }
 
-    private void build() {
-
+    /**
+     * Builds the branch that was cloned into the target directory.
+     */
+    private void build() throws IOException, InterruptedException {
+        String path = dirPath + "/Continuous-Integration/src/main/java/com/ci/ContinuousIntegrationServer.java";
+        String[] arguments = {"javac", "-cp", "servlet-api-2.5.jar:jetty-all-7.0.2.v20100331.jar", path};
```
Comment (jonassavas) [inline src/main/java/com/ci/ContinuousIntegrationServer.java]: I think the build function should simply run ./gradlew build in the root of the cloned repo

**Review by jonassavas — COMMENTED** [review]
Instead of compiling the specific java-file we should run the ./gradlew build command.

**Comment by simonblo** [comment]
The `build` function has been modified now.

**Review by jonassavas — APPROVED** [review]
Looks good to me!

**Pull request merged**